src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -271,15 +271,28 @@
     boolean hasChildren();
 
     /**
-     * Checks for whether the given stack is contained in this Inventory. This
-     * is equivalent to calling <code>!inv.query(stack).hasChildren();</code>
+     * Checks whether the stacks quantity or more of given stack is
+     * contained in this Inventory. This is equivalent to calling
+     * <code>!inv.query(stack).hasChildren();</code> To check if an
+     * inventory contains any amount use {@link #containsAny(ItemStack)}.
      *
      * @param stack The stack to check for
-     * @return True if the stack is present in this list
+     * @return True if the at least the stack stack is present in this inventory
```

Comment:
Something is wrong with this sentence,
Maybe something like this is intended.
`@return true if there are at least the given stack's amount of items present in this inventory.`